daml-lf/spec/daml-lf-1.rst

```diff
@@ -1588,6 +1607,23 @@ for the ``DefTemplate`` rule). ::
   ——————————————————————————————————————————————————————————————— DefException [Daml-LF ≥ 1.14]
     ⊢  'exception' T ↦ { 'message' e }
 
+    Mod:T  ∉  { Mod₁:T₁, …, Modₖ:Tₖ }
+    'interface' (x₁ : T₁) ↦ { 'requires' R₁ , … }  ∈ 〚Ξ〛Mod₁     R₁ ⊆ { Mod₁:T₁, …, Modₖ:Tₖ }
```

Comment:
Can you help me understand your notation here? I would have expected that `requires` points to an interface so `R_1` is an interface. But then you say the interface is a subset of a set of templates?

Reply:
Oh I see, `T_1` is not a template, it’s an interface and `R_1` is the set of all requires. That does make sense but I think it might be clearer if we use `I_1` for interfaces in place of templates.

Reply:
Yep, `R_1` is a set -- it seemed a lot easier to specify this way.
I agree with using `I` for interfaces, let me change that.